doc/source/toolkit/ui.rst

```diff
@@ -2,81 +2,100 @@
 UI reference
 ============
 
-This section describes the **Antenna Wizard**.
-To run the **Antenna Wizard**, you must have a licensed copy of Ansys Electronics
-Desktop (AEDT) installed.
+This section describes how to use the Antenna Toolkit Wizard, which requires an
+installed and licensed copy of AEDT. It assumes that you have already launched the
+wizard from either the AEDT menu or AEDT console. For toolkit installation and wizard
+launching information, see these topics:
 
-Once the **Antenna Wizard** is launched, you can choose settings from first page and connect to an existing
-AEDT session or create a new session.
+- :ref:`install-toolkit-AEDT`
+- :ref:`install_toolkit_console_ui`
 
-.. image:: ../_static/settings.png
-  :width: 800
-  :alt: Settings Tab
+#. On the **AEDT Settings** tab, specify settings for either creating an AEDT session or
+   connecting to an existing AEDT session.
 
-If the toolkit is launched from AEDT, the previous tab does not appear because the toolkit is directly connected to
-the specific AEDT session.
+   .. note::
+      If the Antenna Toolkit Wizard is launched from AEDT, the **AEDT Settings** tab does not appear
+      because the toolkit is directly connected to the specific AEDT session.
 
-Once the toolkit is connected to AEDT, you can load the antenna template from the list **Antennas**.
+   .. image:: ../_static/settings.png
+      :width: 800
+      :alt: Settings tab
 
-.. image:: ../_static/design_connected.png
-  :width: 800
-  :alt: Antenna Toolkit UI, Design Tab
+   The wizard has a progress bar and a logger box, where you can see the status of every operation.
+   A red progress bar means that the toolkit is busy. Every operation must wait until the toolkit is released.
 
-For example, if you select Antennas>Horn>Conical Corrugated, the synthesis option and the antenna template appears.
+   Once the wizard is connected to AEDT, you want to use the **Antennas** list in AEDT to load an antenna
```

Comment:
@Samuelopez-ansys I wasn'tsure where this **Antenna** list was--and had to assume in was in AEDT because I didn't see any such label in the wizard. Please correct if my assumption is wrong.

Reply:
@PipKat It is here in the UI: 
<img width="551" alt="image" src="https://github.com/ansys/pyaedt-toolkits-antenna/assets/85613111/a00f60db-b3fb-4608-8383-cab9a12c1839">